google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineApplicationInfoPanel.java

```diff
@@ -47,35 +46,40 @@
  */
 public class AppEngineApplicationInfoPanel extends JPanel {
```

Comment:
On a second thought, I can look into a way of extracting this into a new PR if you prefer. IIUC, they should be independent changes.